src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseFileService.cs

```diff
@@ -131,6 +132,23 @@ public class ReleaseFileService : IReleaseFileService
                 });
         }
 
+        public async Task<Either<ActionResult, FileInfo>> GetFile(Guid releaseId, Guid fileId)
```

Comment:
You should add a tests for this to `ReleaseFileServiceTests` such as both file and blob exist, file not exists, blob not exists.